GWAS/LMM.ipynb

```diff
@@ -956,7 +956,7 @@
     "parameter: mind_filter = 0.0\n",
     "input: bfile\n",
     "output: f'{cwd}/cache/{bfile:bn}.qc_pass.id', f'{cwd}/cache/{bfile:bn}.qc_pass.snplist' \n",
-    "task: trunk_workers = 1, walltime = walltime, mem = mem, cores = numThreads, tags = f'{step_name}_{_output[0]:bn}'\n",
+    "task: trunk_workers = 1, walltime = '10h', mem = '20G', cores = numThreads, tags = f'{step_name}_{_output[0]:bn}'\n",
```

Comment:
why do we have to hardcode the resource usage here?

Reply:
@gaow I had to hard code this part for it to run on the cluster otherwise I kept getting errors in the job. I didn't no really investigate further than that..but I can PR this version without hard coding that part

Reply:
I know exactly a better way to do it (roughly speaking is you take the min(10hr, walltime) and min(20G , mem) but I dont have the time now to do this very carefully. I suggest putting in **bold** text in the markdown cell before this line to say that this is the minimum requirement based on your experience, and put a link on this particular PR in the text. I'll then accept it. In the future if someone wants to change it to get larger resource they can revisit and impelment what i suggest above.

Reply:
@gaow added your suggestions to pipeline